storage/rocksdb/rdb_datadic.h

```diff
@@ -203,6 +203,22 @@ enum {
   UNPACK_FAILURE = 1,
 };
 
+
+/*
+  Global user-visible data dictionary format version.
+  The server will read the data of any version, but will not write data
+  structures that were introduced after the version in
+  rocksdb_table_dictionary_format.
+  This way, one can keep the on-disk data backward-compatible.
+*/
+const uint ROCKSDB_DATADIC_FORMAT_INITIAL = 1;
+const uint ROCKSDB_DATADIC_FORMAT_CREATE_TIMESTAMP = 2;
+
+// Maximum possible value:
+const uint ROCKSDB_DATADIC_FORMAT_MAX = 2;
+const uint ROCKSDB_DATADIC_FORMAT_DEFAULT = ROCKSDB_DATADIC_FORMAT_MAX;
```

Comment:
Can the default be ROCKSDB_DATADIC_FORMAT_INITIAL instead so that MyRocks and PerconaRocks don't automatically change behaviors? Can you commit a separate change to update it to 2 on the MariaDB branch?